apps/callscreen/style/conference_group_ui.css

```diff
@@ -39,6 +44,9 @@
   bottom: 0;
   left: 0;
 }
+html[dir="rtl"] #group-call-details .hangup-button {
+  background: url('images/handled_call/actionicon_activecall_hangup.png') no-repeat scroll left 1.8rem center / 4rem auto transparent;
+ }
```

Comment:
nit: The closing brace has a space before it, also please leave an empty line between the previous ruleset and this one